.github/workflows/apply_sw4hw_standards.yml

```diff
@@ -0,0 +1,45 @@
+name: Apply SW4HW Standards
+on: 
+  workflow_dispatch:
+
+  pull_request:
+    types: [opened, reopened, synchronize, edited]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  standardize:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: install socat
+        run: |
+          sudo apt install -y socat
+
+      - name: Configure git with qubot
+        run: |
+          git config --global user.name 'The OG Bot Betty McBottington'
+          git config --global user.email 'sw4hw-team@xanadu.ai'
```

Comment:
I wonder if you could comment, @martellato, as to how much of this is needed if we don't plan to have the bot make commits to this repo.

Reply:
We can get rid for sure. I copied these in a loop for all plugins iirc